app/javascript/components/users-dropdown.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <clickable-dropdown
+      :body-title="dropdownTitle"
+      :no-items-message="noUsersMessage"
+      :items="users"
+      @item-clicked="onItemClicked"
+    /><br>
+    <div v-for="(user, index) in selected"
+      :key="user.id"
+    >
+      <button v-on:click="deleteItem(index)">X</button>
+      {{ user.login }}
+    </div><br>
+  </div>
+</template>
+
+<script>
+import ClickableDropdown from './clickable-dropdown';
+
+export default {
+  props: {
+    users: Array
+  },
+  data() {
+    return {
+      dropdownTitle: 'Usuarios',
+      noUsersMessage: 'No se encontraron usuarios',
+      selected: [],
+    };
+  },
+  methods: {
+    onItemClicked({ item }) {
+      if (!this.selected.includes(item)) {
+        this.selected = [...this.selected, item];
+        this.$emit('UpdateSelected', this.selected);
+      }
+    }, 
+    deleteItem({index}){
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
  A space is required before '}'                                                    object-curly-spacing